src/components/App/App.js

```diff
@@ -56,10 +57,15 @@ function App() {
 
   return (
     <div className="App">
-      <Route 
-        exact path="/"
-        render={() => <Wall artworks={wall} />}
```

Comment:
Sounds like we need a component here instead of a render

Reply:
+ Since we're passing down props, it should remain render.  Let me know if you'd like to discuss this further.